features/webdriver-compute-pressure.yml

```
@@ -0,0 +1,10 @@
+name: WebDriver compute pressure
+description: WebDriver Compute Pressure are WebDriver extension commands for creating, deleting, and updating readings for virtual pressure sources (sources that do not depend on underlying hardware or operating system support).
```

More inline with our usual style of "The `entrypoint` [technology] [noun] [verbs] [nouns]".

```suggestion
description: The `CreateVirtualPressureSource` WebDriver command creates artificial sources of compute pressure for testing without the need real device sources of compute pressure.
```